frontend/src/router.ts

```diff
@@ -12,21 +12,20 @@ import Portal from './views/Portal.vue';
 
 import {
   raid as featureFlagRaid,
-  stakeOnly as featureFlagStakeOnly,
   market as featureFlagMarket,
   portal as featureFlagPortal
 } from './feature-flags';
 
 function createRouter() {
-  if (featureFlagStakeOnly) {
-    return new VueRouter({
-      routes: [
-        { path: '/', redirect: 'stake' },
-        { path: '/stake', name: 'select-stake-type', component: SelectStakeType },
-        { path: '/stake/:stakeType', name: 'stake', component: Stake, props: true },
-      ]
-    });
-  }
+  // if (featureFlagStakeOnly) {
```

Comment:
the reason we're intending to remove it is not to leave it as comments in the codebase. please remove the code entirely.

Reply:
Ok, I just thought you guys might want some substitute logic for the logic flows that this flag catches... guess  I can delete them safely then

Reply:
Btw in env.mainnet and testnet.. do I delet the flag?  Coz it says just permanently disabled so Im unsure?

VUE_APP_STAKING_ONLY="0"---- > delete?